integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -706,4 +716,65 @@ class ApplicationsIT extends ITSupport {
         // verify
         assertNotPresent(app.listScopeConsentGrants(), app.getScopeConsentGrant(oAuth2ScopeConsentGrant.getId()))
     }
+
+    @Test
```

Comment:
Add one test with `application/json` only.

Reply:
..or is it better to add a test without “Accept” at all? In this case “application/json” should be added in `applyDefaultRequestHeaders()`

Reply:
yeah, that is fine.